dev-python/cadquery-ocp/cadquery-ocp-7.5.3.0.ebuild

```diff
@@ -0,0 +1,195 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Could we also bump this to EAPI=8 please?

Reply:
I'll try

Reply:
Changed to 8, waiting for the build (`ocp` seems to ignore my ccache by some reason, so it will take some time).

In the meantime, could you enable CI for this PR? `pkgcheck` only complains about overshadowed versions locally, but I'm not sure I'm running it right.